helm/artemis/Chart.yaml

```diff
@@ -0,0 +1,46 @@
+apiVersion: v2
+name: artemis
+description: A Helm chart for Artemis 
+
+# A chart can be either an 'application' or a 'library' chart.
+#
+# Application charts are a collection of templates that can be packaged into versioned archives
+# to be deployed.
+#
+# Library charts provide useful utilities or functions for the chart developer. They're included as
+# a dependency of application charts to inject those utilities and functions into the rendering
+# pipeline. Library charts do not define any templates and therefore cannot be deployed.
+type: application
+
+# This is the chart version. This version number should be incremented each time you make changes
+# to the chart and its templates, including the app version.
+# Versions are expected to follow Semantic Versioning (https://semver.org/)
+#TODO: figure out how and when this is updated
+version: 0.1.2
+
+# This is the version number of the application being deployed. This version number should be
+# incremented each time you make changes to the application. Versions are not expected to
+# follow Semantic Versioning. They should reflect the version the application is using.
+# It is recommended to use it with quotes.
+appVersion: "5.5.7"
```

Comment:
I assume this is the Artemis version? If so, this is a) very outdated and b) who will update this for future releases?

Reply:
Yep, good catch! Forgot to bump it.

Reply:
So this should be the Artemis version that the chart currently comes with by default. For the PR deployments, we edit this variable. This should be updated whenever there is a breaking change in Artemis that requires changes to the values.yaml in the helm chart together with the artemis version in values.yaml and the helm chart value in a SemVer style.

Reply:
Not sure who will maintain the helm chart — though I propose simply bumping it whenever and whoever bumps the artemis-collection since it's mostly a repetitive task.